vite.config.ts

```diff
@@ -17,6 +17,14 @@ export default defineConfig(({ mode }) => {
   const forceLocalStorage = env.FORCE_LOCAL_STORAGE === 'true'
   const privateJrPlatform = env.PRIVATE_JR_PLATFORM === 'true'
 
+  if (scrivitoTenantIsMissing(env)) {
+    throw new Error(
+      'Environment variable "SCRIVITO_TENANT" is not defined!' +
+        ' Check if the ".env" file with a proper SCRIVITO_TENANT is set.' +
+        ' See ".env.example" for an example.',
+    )
```

Comment:
* [ ] FYI The error message is not accurate given the implementation.

Reply:
Do you mean, because it can also look into `.env.local`?

Reply:
Instead of not passing SCRIVITO_TENANT maybe the user wanted to set PRIVATE_JR_PLATFORM?

Reply:
No - `PRIVATE_JR_PLATFORM` is a private variable, that is only intended for "us" scrivito developer who are running a multi tenancy app. No-one outside of our team should use `PRIVATE_JR_PLATFORM`.